### DISCUSSION TITLE
When using image-webpack-loader to compress images, the following error is reported

### DISCUSSION THREAD

**Discussion by X-jc** [web-infra-dev/rsbuild#3867] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/c59f16f1-5649-4649-ab1f-508fa958d7bb)

![image](https://github.com/user-attachments/assets/488a03ca-f3c0-4a7c-a52b-68a8d1586d79)
求解答

**Comment by 9aoy** [comment, marked as answer]
https://rspack.dev/config/module#ruletest or https://rsbuild.dev/config/tools/bundler-chain#chain_idrule
